modules/EVSE/EvseManager/Charger.cpp

```diff
@@ -1077,18 +1077,20 @@ float Charger::ampere_to_duty_cycle(float ampere) {
 }
 
 bool Charger::set_max_current(float c, std::chrono::time_point<std::chrono::steady_clock> validUntil) {
-    if (c >= -CHARGER_ABSOLUTE_MAX_CURRENT and c <= CHARGER_ABSOLUTE_MAX_CURRENT) {
+    float c_abs{std::fabs(c)};
```

Comment:
I dont like the variable name `c_abs` but I could not find a better alternative :sweat_smile: